app/views/posts/show.html.erb

```diff
@@ -26,4 +26,8 @@
       <% end %>
       </ul>
     </div>
+    <%= form_with model: @like,  url: "#{post.id}/like_toggle", method: :post , local: true do |form|%>
+      <a href=<%="#{post.id}/comments/new"%>>Add comment</a>
+      <%= form.submit "Like/Unlike this post" %>
+    <% end %>
```

Comment:
- [ ] These two buttons need not be within a form when we consider structuring our page semantically. They can just be two separate buttons that lead to some path.

  Did you have any special reasons for wrapping these two in a form?
  
  That aside, currently, a user cannot create a like when the like button is clicked. Even though you've defined a likes path in the routes, there is no dedicated controller that handles the creation of a like to a post.
  
  Please add the `LikesController` and use it to enable liking a post.